build/dependencies.yaml

```diff
@@ -151,7 +151,7 @@ dependencies:
     - path: build/pause/Makefile
       match: TAG\s*\?=
 
-  - name: "k8s.gcr.io/pause: dependents"
+  - name: "registry.k8s.io/pause: dependents"
```

Comment:
Sorry, but I'm a little confused here.
The value in the following `match` field is still `k8s.gcr.io`, which looks inconsistent with the `name` description.
Do we really need to make this change in this PR?

Reply:
right, this should have only matched against kubeadm paths

Reply:
updated this part